vendor/grammars/blade/CHANGELOG.md

```diff
@@ -0,0 +1,99 @@
+# Change Log
```

Comment:
\*sigh* @jbrooksuk , you may actually want to repeat the steps [I mentioned here](https://github.com/github/linguist/pull/3008#issuecomment-230452322). Both in your local filesystem and on GitHub.

You forgot to delete the old repo, it seems (which was sitting at `vendor/grammars/blade`). Burn the entire folder, delete the repo, refork, and [follow these steps very carefully](https://github.com/github/linguist/blob/master/CONTRIBUTING.md#adding-a-language).